File: NewProject/wwwroot/js/Class/Spsyz.js
//商品适应症
export default class Spsyz {
    constructor() {
        //适应症ID
        this.syzbh = "";
        //标准适应症名称
        this.syzmc = "";
    }
    addData(obj) {
        this.syzbh = obj.syzbh;
        this.syzmc = obj.syzmc;
    }
}

File: NewProject/Models/SPSYZ.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;


namespace NewProject.Models
{
    //适应症
    public class SPSYZ : Table
    { 
        [Key]
        public int GUID { get; set; }
        [Display(Name = "适应症ID")]
        public string SYZBH { get; set; }
        [Display(Name = "产品ID")]
        public int SPBH { get; set; }
        [Display(Name = "标准适应症名称")]
        public string SYZMC { get; set; }
    }
}
